### PR TITLE
add zstd and webp support

### PR DESCRIPTION
**Pull request by bitner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -58,6 +58,29 @@ RUN mkdir -p /tmp/libjpeg-turbo \
   && cmake -G"Unix Makefiles" -DCMAKE_INSTALL_PREFIX=/opt . \
   && make -j $(nproc) install
 
+## webp
+
+RUN mkdir -p /tmp/webp \
+    && cd /tmp/webp \
+    && curl -f -L -O https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.0.0.tar.gz \
```
Comment (sethfitz) [inline Dockerfile]: Would you mind sticking the version number in the top as an `ARG`?

File: Dockerfile [hunk 2]
```diff
@@ -58,6 +58,29 @@ RUN mkdir -p /tmp/libjpeg-turbo \
...
+    && cd libwebp-1.0.0 \
+    && CFLAGS="-O2" ./configure --prefix=/opt \
+    && make \
+    && make install
+
+## libszstd
+
+RUN mkdir -p /tmp/libszstd \
+    && cd /tmp/libszstd \
+    && curl -f -L -O https://github.com/facebook/zstd/archive/dev.zip \
```
Comment (sethfitz) [inline Dockerfile]: Is there a reason you're using the `dev` branch instead of a tagged release?

**Review by sethfitz — CHANGES_REQUESTED** [review]
Awesome! A couple small requests, if you please.

File: Dockerfile (hunk 1 above)
Comment (bitner) [reply Dockerfile]: no problem

File: Dockerfile (hunk 2 above)
Comment (bitner) [reply Dockerfile]: it was just a modified copy paste from an existing build I had. modified it to use 1.3.8

**Review by sethfitz — APPROVED** [review]

**Comment by sethfitz** [comment]
Thank you!

**Pull request merged**